API-6944: approve_agent: update required scopes (#734)
livechat/livechat-public-docs: content/management/configuration-api/index.mdx

|                 |                                                                       |
| --------------- | --------------------------------------------------------------------- |
| Method URL      | `https://api.livechatinc.com/v3.2/configuration/action/approve_agent` |
| Required scopes | `agents-all:rw` `subscription_manage` __*__                           |

> __*__ `subscription_manage` is a private scope, it's not possible to obtain it via Developers Platform.

Reviewer: ```suggestion
__*__ `subscription_manage` is a private scope; it's not possible to configure it in Developer Console.
```
